src/tools/qt-gui/org.libelektra.elektra-qt-editor.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Elektra Qt Editor
+GenericName=Qt Configuration Editor for Elektra
```

Comment:
Ok going to drop the Qt. It probably should not be in the generic name anyway.  The line is based on the one from dconf-editor desktop file.